svg/shapes/ellipse-04.svg

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg xmlns="http://www.w3.org/2000/svg" width="100" height="100">
+  <metadata>
+    <link xmlns="http://www.w3.org/1999/xhtml" rel="help" href="http://www.w3.org/TR/SVG2/shapes.html#RectElement"/>
+    <link xmlns="http://www.w3.org/1999/xhtml" rel="match" href="ellipse-04-ref.svg"/>
+    <meta xmlns="http://www.w3.org/1999/xhtml" name="assert" content="rx and ry auto disables rendering."/>
+  </metadata>
+  <style>
+    ellipse {
+      rx: auto;
+      ry: auto;
+    }
+  </style>
+  <ellipse cx="50" cy="50" fill="none" stroke="orange" stroke-width="5"/>
```

Comment:
Maybe here it'd make sense to actually use 'red' as the stroke color (to clearly show if there's an error.)